2996_reflection/reflection.md

```diff
@@ -3575,7 +3577,7 @@ constexpr int f = template [:^^e:]<^^S::a>;  // splice-expression
 
 [The implicit transformation ([expr.prim.id]{.sref}) whereby an `$id-expression$` denoting a non-static member becomes a class member access does not apply to a `$splice-expression$`.]{.note}
 
-[#]{.pnum} For a `$splice-expression$` that designates a function or function template, overload resolution is performed ([over.match]{.sref}, [temp.over]{.sref}) from an initial set of candidate functions containing only that entity designated by the `$splice-expression$`. The best viable function selected by overload resolution is considered to be designated in a manner exempt from access rules.
+[#]{.pnum} For a `$splice-expression$` that designates a function or function template, overload resolution is performed ([over.match]{.sref}, [temp.over]{.sref}) from an initial set of candidate functions containing only the entity designated by the `$splice-expression$`. The best viable function selected by overload resolution is _designated in a manner exempt from access rules_.
```

Comment:
Stray `_` in front of designated?

Reply:
Nope, "designated in a manner exempt from access rules" is the term being introduced.

Reply:
Ah just couldn't see the other `_`